.github/workflows/elixir.yml

```diff
@@ -50,4 +48,4 @@ jobs:
         mix deps.get
 
     - name: Run tests
-      run: mix test
+      run: mix test --timeout 120000
```

Comment:
What's with this timeout?

Reply:
Since we removed imageflow, transforming images takes much longer.

Reply:
can you then open an issue so this doesn't go forgotten then? it feels like somthing that shouldn't be here (even though I know it doesn't really have real impact)

Reply:
Done https://github.com/still-ex/still/issues/127